prowler/providers/kubernetes/services/rbac/rbac_client.py

```diff
@@ -0,0 +1,4 @@
+from prowler.providers.common.common import global_provider
+from prowler.providers.kubernetes.services.rbac.rbac_service import Rbac
+
+rbac_client = Rbac(global_provider)
```

Comment:
```suggestion
rbac_client = RBAC(global_provider)
```

Reply:
The Kubernetes client calls it Rbac https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/RbacAuthorizationV1Api.md